pkg/kubectl/cmd/util/env/doc.go

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
The copyright boilerplate needs to have a newline here (and below).

The one in `env_parse_test.go` is right. :)